tests/test_postgres.py

```diff
@@ -1748,6 +1748,26 @@ def test_has_monitoring_privileges_exception(self, conn_mock, mock_is_superuser)
         # THEN a None value was returned
         assert has_monitoring is None
 
+    @patch("barman.postgres.PostgreSQLConnection.has_backup_privileges")
+    @patch("barman.postgres.PostgreSQLConnection.connect")
+    def test_current_size(self, conn_mock, has_backup_privileges_mock):
+        """
+        Test the current_size method
+        """
+        size = 2048
+        has_backup_privileges_mock.return_value = True
+        # Build a server
+        server = build_real_server()
+        conn_mock.return_value.server_version = 110000
```

Comment:
I suppose this was copied from other mocks (didn't check), but shouldn't we test with newer versions? Maybe it's ok/better to test with the oldest supported version.

P.S.: we should have named this sever_version_num which is the field in pg_settings. But that's outside the scope of this PR

Reply:
We can remove the server_version! It is not mentioned in the method itself! I got it wrongly from the xlog_segment_size property! I will fix